devito/arch/archinfo.py

```diff
@@ -19,6 +19,8 @@
            'get_cuda_path', 'get_hip_path', 'check_cuda_runtime', 'get_m1_llvm_path',
            'Platform', 'Cpu64', 'Intel64', 'IntelSkylake', 'Amd', 'Arm', 'Power',
            'Device', 'NvidiaDevice', 'AmdDevice', 'IntelDevice',
+           # Generic
+           'CPU64',
```

Comment:
SO both CPU and Cpu ?

Reply:
CPU64 = Cpu64('cpu64')

in archinfo

Reply:
Agrred, CPU64 + Cpu64 seems like there shound't need both